File: solutions/cpp/log-levels/1/log_levels.cpp
#include <string>

namespace log_line {
std::string message(std::string line) {
    // return the message
    return line.substr(9);
}

std::string log_level(std::string line) {
    // return the log level
    return line.substr(1, 6);
} 

std::string reformat(std::string line) {
    // return the reformatted message
}
return line.substr(9) + " (" + line.substr(1, 6) + ")";
}  // namespace log_line

File: solutions/java/log-levels/1/src/main/java/LogLevels.java
public class LogLevels {

    public static String message(String logLine) {
    String[] log = logLine.split(":");
    return log[1].trim();
    }

    public static String logLevel(String logLine) {
    String[] log = logLine.split("\\[");
    String[] line = log[1].split("\\]");
    String retu = line[0];
    return retu.toLowerCase();
    }

    public static String reformat(String logLine) {
    String concat = message(logLine) + " " + "(" + logLevel(logLine) + ")";
    return concat;
    }
}
